### PR TITLE
Add volunteers

### PR DESCRIPTION
**Pull request by ajpinedam**
Adding two more volunteers to our team.

@ZevEisenberg  in the template if there is no social information should this be empty or not present at all?

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -208,3 +208,22 @@
   type: mc
   social:
     - {name: "twitter", link: "https://twitter.com/garricn"}
+
+-
+  id: "17"
```
Comment (ZevEisenberg) [inline _data/team.yml]: @ajpinedam please make sure these ID numbers are unique inside this file. There is already an `id: 17`.

File: _data/team.yml [hunk 2]
```diff
@@ -208,3 +208,22 @@
...
+    - {name: "twitter", link: "https://twitter.com/skelegram"}
+
+-
+  id: "18"
+  name: "Ari"
+  surname: "Roshko"
+  title: "Team Player"
+  thumbnailUrl: AriRoshko.jpg
+  type: volunteer
+  social: []
```
Comment (ZevEisenberg) [inline _data/team.yml]: Please add a new line at the end of the file.

**Review by ZevEisenberg — CHANGES_REQUESTED** [review]

File: _data/team.yml (hunk 1 above)
Comment (ajpinedam) [reply _data/team.yml]: @ZevEisenberg 9 and 10 are also repeated you want me to fix these too?

Comment (ZevEisenberg) [reply _data/team.yml]: Oh, yes, please!

**Review by ZevEisenberg — APPROVED** [review]

**Pull request merged**